### ISSUE TITLE
`app.options` is undefined when using in an addon

### ISSUE THREAD

**Issue by wagenet** [pangratz/ember-cli-htmlbars-inline-precompile#4]
Issue is at https://github.com/pangratz/ember-cli-htmlbars-inline-precompile/blob/master/index.js#L13

**Comment by pangratz** [comment]
Hey @wagenet, thanks for reporting. I have difficulties reproducing the issue though. Which Ember-CLI version does the addon use?

With the following steps I couldn't reproduce:

``` sh
$ ember version
version: 0.2.3
node: 0.12.2
npm: 2.7.6

$ ember addon test-addon
$ cd test-addon

$ ember install ember-cli-htmlbars-inline-precompile

$ echo "import Ember from 'ember';
export default Ember.Component.extend();" > addon/components/my-component.js

$ echo "import hbs from 'htmlbars-inline-precompile';
import { moduleForComponent, test } from 'ember-qunit';

moduleForComponent('my-component');

test('it renders', function(assert) {
  var component = this.subject({
    template: hbs`<span>hello</span>`
  });

  assert.equal(this.$().html().trim(), '<span>hello</span>');
});" > tests/unit/components/my-component-test.js

$ ember test
version: 0.2.3
Built project successfully. Stored in "/private/tmp/test-addon/tmp/class-tests_dist-bpiJ3JNg.tmp".

...

1..8
# tests 8
# pass  8
# fail  0

# ok
```

**Comment by wagenet** [comment]
I ran into the issue when using the addon that depended on the hbs compiler in another ember app.

**Comment by pangratz** [comment]
Ah, ok. I'll try that scenario then ...

**Comment by mrosenberg** [comment]
I'm running into the same issue. Is there a work around until this is merged?

```
$ ember version
version: 1.13.8
node: 0.12.2
npm: 2.13.4
os: darwin x64

$ ember serve
version: 1.13.8
Cannot read property 'babel' of undefined
TypeError: Cannot read property 'babel' of undefined
    at Class.module.exports.included (/Users/matthewrosenberg/repositories/ember-charts/node_modules/ember-cli-htmlbars-inline-precompile/index.js:13:36)
    at /Users/matthewrosenberg/repositories/ember-charts/node_modules/ember-cli/lib/models/addon.js:253:32
    at Array.map (native)
    at Class.eachAddonInvoke (/Users/matthewrosenberg/repositories/ember-charts/node_modules/ember-cli/lib/models/addon.js:251:22)
    at Class.Addon.included (/Users/matthewrosenberg/repositories/ember-charts/node_modules/ember-cli/lib/models/addon.js:357:8)
    at EmberAddon.<anonymous> (/Users/matthewrosenberg/repositories/ember-charts/node_modules/ember-cli/lib/broccoli/ember-app.js:328:15)
    at Array.filter (native)
    at EmberAddon.EmberApp._notifyAddonIncluded (/Users/matthewrosenberg/repositories/ember-charts/node_modules/ember-cli/lib/broccoli/ember-app.js:323:45)
    at EmberAddon.EmberApp [as appConstructor] (/Users/matthewrosenberg/repositories/ember-charts/node_modules/ember-cli/lib/broccoli/ember-app.js:118:8)
    at new EmberAddon (/Users/matthewrosenberg/repositories/ember-charts/node_modules/ember-cli/lib/broccoli/ember-addon.js:39:8)
```

**Comment by pangratz** [comment]
Okay, I initially thought that this might be solved via #9. But since that PR is not quite finished yet and the problem still exists, I went ahead and merged the changes proposed by @wagenet in #5, which are released in `v0.3.0`.

@mrosenberg can you test/verify that the latest release works for you?

**Comment by mrosenberg** [comment]
I created a new addon and copied over my code this morning with no issues. I'm not sure what created the conflict but it seems to have been resolved.
